source/connect/client.txt

```diff
@@ -0,0 +1,110 @@
+
+- A **connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- A **MongoDB\\Client** object, which creates the connection to the MongoDB deployment
+  and lets you perform operations on it.
+
+You can also use either of these components to customize the way the {+driver-short+} behaves
+while connected to MongoDB.
+
+This guide shows you how to create a connection string and use a ``MongoDB\Client`` object
+to connect to MongoDB.
+
+.. _php-connection-uri:
+
+Connection URI
+--------------
+
+A standard connection string includes the following components:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Component
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
+
+   * - ``username:password``
+
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``.
+       For more information about the ``authSource`` connection option, see
+       :ref:`php-auth`. 
+
+   * - ``host[:port]``
+
+     - Required. The host and optional port number where MongoDB is running. If you don't
+       include the port number, the driver uses the default port, ``27017``.
+
+   * - ``/defaultauthdb``
+     
+     - Optional. The authentication database to use if the
+       connection string includes ``username:password@``
+       authentication credentials but not the ``authSource`` option. If you don't include 
+       this component, the client authenticates the user against the ``admin`` database.
+
+   * - ``?<options>``
+
+     - Optional. A query string that specifies connection-specific
+       options as ``<name>=<value>`` pairs. See
+       :ref:`php-connection-options` for a full description of
+       these options.
+
+For more information about creating a connection string, see
+:manual:`Connection Strings </reference/connection-string>` in the
+MongoDB Server documentation.
+
+Create a ``MongoDB\Client``
+---------------------------
+
+To create a connection to MongoDB, construct an instance of the ``MongoDB\Client`` class,
+passing the connection URI as a string to the constructor.
+
+In the following example, the library uses a sample connection URI to connect to a MongoDB 
+deployment on port ``27017`` of ``localhost``:
+
+.. literalinclude:: /includes/connect/client.php
+   :language: php
+   :copyable: true
+
+.. tip:: Reuse Your Client
+
+   Because each ``MongoDB\Client`` object represents a pool of connections to the
+   database, most applications require only a single instance of
+   ``MongoDB\Client``, even across multiple requests.
+
+API Documentation
+-----------------
+
+To learn more about creating a ``MongoDB\Client`` object in the {+driver-short+},
+see the following API documentation:
+
+- :ref:`MongoDB\Client <php-api-mongodbclient>`
```

Comment:
I: this is presented as an API link, but I see a ref, is this supposed to link here https://www.mongodb.com/docs/php-library/current/reference/class/MongoDBClient/?

Reply:
yes, the PHP docs host their own API docs

Reply:
Hmm, inter-repo refs should work. Maybe you can double check the snooty-toml to make sure its included in the intersphinx?

Reply:
Good thought, but this is within the same repo. Looks like my addition of the ref at the top of that file wasn't recognized, even though I had committed the change (I removed the last line of the file just so VS Code would let me save/commit again).

Reply:
Looks to be working now!